Week2/task4.js

```diff
@@ -0,0 +1,16 @@
+console.log(`More JavaScript`);
+console.log(`
+Task 4 `); // a function which prints the color and a type of the vehicle
+
+function vehicleType(color, code) {
+  if (code === 1) {
+    code = 'car';
+  } else if (code === 2) {
+    code = 'motorbike';
+  } else {
+    code = 'but vehicle type is unknown';
+  }
+  console.log('a ' + color + ' ' + code);
+}
+
+vehicleType('blue', 3);
```

Comment:
Did you try this line? It returns `a blue but vehicle type is unknown`. The code is not wrong but the result is a bit weird.

Reply:
Since we have only code 1 and 2 I try to put extra explanation, for the cases where the code will be different than 1 and 2. For 1 and 2 it works.